src/protocols/vnc/vnc.c

```diff
@@ -416,6 +416,20 @@ void* guac_vnc_client_thread(void* data) {
     }
 #endif
 
+    /* Disable remote console (Server input) */
+    if (settings->disable_server_input) {
+        rfbSetServerInputMsg msg;
+        msg.type = rfbSetServerInput;
+        msg.status = 1;
+        msg.pad = 0;
+
+        if (WriteToRFBServer(rfb_client, (char*)&msg, sz_rfbSetServerInputMsg))
+            guac_client_log(client, GUAC_LOG_DEBUG, "Successfully sent request to disable server input.");
+
+	else
+            guac_client_log(client, GUAC_LOG_WARNING, "Failed to send request to disable server input.");
```

Comment:
A tab snuck in here an is causing the `else` to appear misaligned:

![Screenshot of misaligned else due to tab](https://github.com/apache/guacamole-server/assets/4632905/fa9cdb3b-b096-46bf-a462-dbfead2d913c)

Reply:
Woops. Fixed via rebase.